soc/nordic/common/vpr/Kconfig.defconfig

```diff
@@ -1,9 +1,6 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-config RISCV_MCAUSE_EXCEPTION_MASK
-	default 0xFFF
```

Comment:
are you sure?

![image](https://github.com/zephyrproject-rtos/zephyr/assets/25011557/7f05b17c-af8c-495e-b1e0-ac84e6048edf)

Reply:
Sorry You are right, this is not an issue., removing PR